Tails specific: don't bother cleaning up interface status, we are amnesic.

--- chroot.orig/etc/init.d/ifupdown	2012-09-24 10:05:13.021047615 +0200
+++ chroot/etc/init.d/ifupdown	2012-09-24 10:47:23.717869294 +0200
@@ -7,1 +7,1 @@
-# Default-Stop:      0 6
+# Default-Stop:
